### PR TITLE
Add Dockerfile for sequence diagram generation

### PR DESCRIPTION
**Pull request by jbeda**

### CODE REVIEW

File: docs/design/clustering/Makefile [hunk 1]
```diff
@@ -14,3 +14,16 @@ $(FONT):
```
Comment (roberthbailey) [inline docs/design/clustering/Makefile]: This doesn't seem to have your last change where you fixed the curl target (removing the .ttf extension).

**Comment by roberthbailey** [comment]
LGTM other than needing a rebase.

File: docs/design/clustering/Makefile (hunk 1 above)
Comment (jbeda) [inline docs/design/clustering/Makefile]: Yeah -- didn't plan these together so kept them in separate branches.

**Comment by jbeda** [comment]
Done now -- merging.  Thanks!

**Pull request merged**